components/02-elements/information/information.hbs

```diff
@@ -0,0 +1,11 @@
+{{#each informations }}
+    <div class="information {{ class }}">
+        <h5 class="heading heading--fifth-level information__title">
+            {{ title }}
+        </h5>
+
+        <span class="information__text">
+            {{ text }}
+        </span>
```

Comment:
Why you use `<span>` and no for example `<p>` if it should be an info text?

Reply:
dunno 🤷‍♂️ changed for now